tests/performance/Makefile

```diff
@@ -6,8 +6,8 @@ load-testing/charts/plutono:
 	curl -sL https://github.com/credativ/helmcharts/releases/download/plutono-0.1.0/plutono-0.1.0.tgz > plutono.tgz
 	tar -xf plutono.tgz -C load-testing/charts
 
-deploy-helm: load-testing/charts/plutono
-	kubectl wait --kubeconfig "${GARDENER_KUBECONFIG}" --for=jsonpath='{.status.lastOperation.state}'=Succeeded --timeout=600s "shoots/${CLUSTER_NAME}"
+deploy-helm:
+	#kubectl wait --kubeconfig "${GARDENER_KUBECONFIG}" --for=jsonpath='{.status.lastOperation.state}'=Succeeded --timeout=600s "shoots/${CLUSTER_NAME}"
```

Comment:
Why is this still commented? Do we want to remove it?

Reply:
Good catch, it was needed for my local tests.